src/components/Poll.tsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from "react";
+import "../styles/poll.css";
+
+// Ratio button vote options array
+const voteOptions = [
```

Comment:
Would it be a good idea to have a reusable RadioInput.tsx component that I can import to my Poll.tsx component? 
<RadioInput value={"JavaScript"} label={"JavaScript"} />
<RadioInput value={"Python"} label={"Python"} />
<RadioInput value={"Go"} label={"Go"} />

I have been exploring that option but wondered if it would conflict with the idea of using props in the HomePage.tsx Poll component.